coredns/README.md

```diff
@@ -2,20 +2,20 @@
 
 ## Overview
 
-Get metrics from CoreDNS in real time to visualize and monitor DNS failures and cache hits/misses.
+Get metrics from CoreDNS in real time to visualize and monitor DNS failures and cache hits or misses.
 
 ## Setup
 
-### Installation
 
-The CoreDNS check is included in the [Datadog Agent][1] package, so you don't need to install anything else on your servers.
+Starting with version 1.11.0, this OpenMetrics-based integration has a latest mode (`use_openmetrics`: true) and a legacy mode (`use_openmetrics`: false). To get all the most up-to-date features, Datadog recommends enabling the latest mode. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][24].
```

Comment:
This is another one of those where `openmetrics_endpoint` is the setting that activates latest mode, as in https://github.com/DataDog/integrations-core/pull/15073#discussion_r1317050535

Reply:
```suggestion
Starting with version 1.11.0, this OpenMetrics-based integration has a latest mode (enabled by setting `openmetrics_endpoint` to point to the target endpoint) and a legacy mode (enabled by setting `prometheus_url` instead). To get all the most up-to-date features, Datadog recommends enabling the latest mode. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][24].
```